packages/schema/parse/src/extract/imported-query-types.ts

```diff
@@ -139,9 +141,18 @@ const visitorEnter = (typeInfo: TypeInfo, state: State) => ({
 
     if (method && argument) {
       // Argument value
-      argument.scalar = createScalarDefinition(
-        argument.name, modifier + node.name.value, state.nonNullType
-      );
+      if (
```

Comment:
Currently, imported query logic and query logic for extracing some types is the same so we should
probably extract that somewhere